test/IntegrationTests/MSTest.Acceptance.IntegrationTests/TestFilterTests.cs

```diff
@@ -33,9 +33,10 @@ public async Task DiscoverTestsWithFilter_UsingTestProperty_FilteredTests(string
         TestHostResult testHostResult = await testHost.ExecuteAsync("--filter tree=one --list-tests");
 
         testHostResult.AssertExitCodeIs(ExitCodes.Success);
-        testHostResult.AssertOutputContains("""
-The following Tests are available:
+        testHostResult.AssertOutputMatchesRegex("""
+Discovered 1 tests in assembly - .*\.dll \(net.+\|.+\)
 Test2
+Discovered 1 tests\.$
```

Comment:
Is this line equivalent to the summary for the run?

Reply:
yes, why do you ask?

Reply:
And duration

Reply:
Maybe something like `Test discovery summary: Found X tests - Asm (tfm|arch)`

Reply:
Isn't it the same assumption (good or bad) for the run?